crates/sui-graphql-rpc/src/extensions/timeout.rs

```diff
@@ -9,49 +9,32 @@ use std::sync::Arc;
 use std::time::Duration;
 use tokio::time::timeout;
 
-// 10s
-const DEFAULT_REQUEST_TIMEOUT: Duration = Duration::from_millis(20_000);
-
-#[derive(Clone, Debug, Copy)]
-pub(crate) struct TimeoutConfig {
-    pub request_timeout: Duration,
-}
-
-impl Default for TimeoutConfig {
-    fn default() -> Self {
-        Self {
-            request_timeout: DEFAULT_REQUEST_TIMEOUT,
-        }
-    }
-}
+use crate::config::ServiceConfig;
 
 #[derive(Clone, Debug, Default)]
-pub(crate) struct Timeout {
-    pub config: TimeoutConfig,
-}
+pub(crate) struct Timeout;
 
 impl ExtensionFactory for Timeout {
     fn create(&self) -> Arc<dyn Extension> {
-        Arc::new(TimeoutExtension {
-            config: self.config,
-        })
+        Arc::new(Timeout {})
```

Comment:
nit: I think the {} are unnecessary here?

```suggestion
        Arc::new(Timeout)
```